asdf/tags/wcs/wcs.py

```diff
@@ -295,3 +314,47 @@ def assert_equal(cls, old, new):
         assert old.name == new.name
         for old_frame, new_frame in zip(old.frames, new.frames):
             helpers.assert_tree_match(old_frame, new_frame)
+
```

Comment:
@philhodge You are right. But this here is an update to the `Galactocentric`  frame in astropy.coordinates which is now initialized by an object `ICRSCoordinates`. So asdf needs to know how to serizlize it.